README.md

```diff
@@ -13,5 +13,101 @@ Most of performed methods are logged using NLog, so you can easily see a history
+>   - Your application consists of several Windows;
+>   - You need to interact with more than one application;
+>   - You need to perform actions before/after the main application run (e.g. reinstall, update).
+
+8. From the PageObject perspective, each Form consists of elements on it (e.g. Buttons, TextBox, Labels and so on). 
+To interact with elements, on your form class create fields of type IButton, ITextBox, ILabel, and initialize them using the ElementFactory. Created elements have a various methods to interact with them. We recommend to combine actions into a business-level methods:
+
+```csharp
+
+using Aquality.Selenium.Core.Configurations;
+using Aquality.WinAppDriver.Applications;
+using Aquality.WinAppDriver.Elements.Interfaces;
+using Aquality.WinAppDriver.Forms;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Appium;
+
+namespace CalcApp.Forms
+{
+    public class CalculatorForm : Form
+    {
+        private readonly ITextBox LeftArgumentTextBox;
+        private readonly IButton OneButton;
+        private readonly IButton TwoButton;
+        private readonly IButton PlusButton;
+        private readonly IButton EqualsButton;
+        private readonly ILabel ResultsLabel;
+
+        public CalculatorForm() : base(By.Name("Day Maxi Calc  v.1.5 Freeware"), "Calculator")
+        {
+            LeftArgumentTextBox = ElementFactory.GetTextBox(MobileBy.AccessibilityId("50"), "Left Argument");
+            OneButton = ElementFactory.GetButton(By.Name("1"), "1");
+            TwoButton = RelativeElementFactory.GetButton(By.Name("2"), "2");
+            PlusButton = ElementFactory.FindChildElement<IButton>(this, By.Name("+"), "+");
+            EqualsButton = FindChildElement<IButton>(By.Name("="), "=");
+            ResultsLabel = ElementFactory.GetLabel(By.XPath("//*[@AutomationId='48']"), "Results bar");
+        }
+
+        private ITimeoutConfiguration TimeoutConfiguration => AqualityServices.Get<ITimeoutConfiguration>();
+
+        public int CountOnePlusTwo()
+        {
+            const string expectedRightArgument = "1";
+            OneButton.Click();
+            ConditionalWait.WaitForTrue(() => LeftArgumentTextBox.Value == expectedRightArgument, 
+                timeout: TimeoutConfiguration.Condition,
+                message: $"Right argument should become '{expectedRightArgument}' after click on {OneButton.Name}");
+
+            PlusButton.MouseActions.DoubleClick();
+            TwoButton.MouseActions.Click();
+            EqualsButton.State.WaitForClickable();
+            EqualsButton.Click();
+            return int.Parse(ResultsLabel.Text);
+        }
+    }
+}
+```
+
+
+
+Please take a look at more examples in our tests at https://github.com/aquality-automation/aquality-winappdriver-dotnet/tree/master/Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Forms.
```

Comment:
is it necessary '/Forms' at the end of the link?

Reply:
If we're talking about the pageObjects, then yes it is